packages/libsql/src/lib/LibSqlClient.ts

```diff
@@ -0,0 +1,203 @@
+import { randomUUID } from "crypto"
+import type { Client, Transaction } from "@libsql/client"
+import { IEventData, ITimelineEvent } from "@equinox-js/core"
+
+type LibSqlWriteResult =
+  | { type: "Written"; position: bigint; snapshot_etag?: string }
+  | { type: "ConflictUnknown" }
+export type Format = string
+
+type Trx = {
+  execute: Client['execute']
+}
+
+export class LibSqlWriter {
+  constructor(private readonly client: Client) {}
+
+  /**
+   * Reads the current version of the stream
```

Comment:
remove L18, making the important bit a one liner - maybe it goes in a header overview comment?
Looking forward to it being fixed in the messagedb impls :P